deployments/setup-fork.ts

```diff
@@ -96,8 +111,32 @@ const fundAccounts = async () => {
         }
     ];
 
+    if (DEV_ADDRESSES === undefined || DEV_ADDRESSES === '') {
```

Comment:
It also could be `[]`. You can use lodash's `isEmpty()` to check for all the cases